internal/pkg/scorecard/scorecard.go

```diff
@@ -310,7 +314,59 @@ func runTests() ([]TestSuite, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to merge test suite results: %v", err)
 	}
-	return suites, nil
+	for _, suite := range suites {
+		// convert to ScorecardOutput format
+		// will add log when basic and olm tests are separated into plugins
+		pluginResults = append(pluginResults, TestSuitesToScorecardOutput([]TestSuite{suite}, ""))
+	}
+	// Run plugins
+	pluginDir := viper.GetString(PluginDirOpt)
+	if dir, err := os.Stat(pluginDir); err != nil || !dir.IsDir() {
+		log.Warnf("Plugin directory not found; skipping plugin tests")
```

Comment:
Also any reason why we're not logging the full error? It'll help with users debug an incorrect path if the error is more explicit:
```Go
	log.Warnf("Plugin directory not found; skipping plugin tests: %v", err)
```